apps/src/templates/teacherDashboard/EditSectionForm.jsx

```diff
@@ -24,31 +24,7 @@ import ConfirmHiddenAssignment from '../courseOverview/ConfirmHiddenAssignment';
 import {SectionLoginType} from '@cdo/apps/util/sharedConstants';
 import firehoseClient from '@cdo/apps/lib/util/firehose';
 
-const style = {
```

Comment:
I think because this is named `style` and not `styles` it wasn't moved down automatically, so I'm moving it down while I'm here.